styles/images.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Anurag Sharma">
+    <title>images</title>
+    <script src="https://kit.fontawesome.com/f45a42e7ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./utility.css">
+
+
+</head>
+
+<body>
+    <!-- responsive image  -->
+    <img src="https://images.unsplash.com/photo-1642726197626-f03ef77f13cf?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80"
+        alt="responsive-image" class="image-res" />
```

Comment:
try adding better name to alt tag..we know its a img... this tag is meant to add meaning to img